lib/core/form/components/widgets/core/form-field.model.ts

```diff
@@ -76,6 +76,7 @@ export class FormFieldModel extends FormWidgetModel {
     rule?: FormFieldRule;
     selectLoggedUser: boolean;
     groupsRestriction: string[];
+    compactForm: boolean = false;
```

Comment:
Better to not use compact as a definition for this functionality. The compact is actually another thing related to form density present in another functionality https://alfresco.atlassian.net/browse/AAE-6939